content/en/apps/reference/forms/contact.md

```diff
@@ -7,39 +7,76 @@ description: >
 relevantLinks: >
   docs/apps/features/contacts
   docs/apps/concepts/hierarchies
+relatedContent: >
+  apps/guides/forms/additional-docs
+  apps/guides/forms/multimedia
+  apps/guides/forms/app-form-sms
 keywords: hierarchy contacts contact-forms
 ---
 
-Contact forms are used for the creation and editing of each contact. The type associated to people is `person`, and there are three types of places corresponding to levels in the app: `district_hospital` > `health_center` > `clinic`. The display names for the place levels can be changed via [custom translations]({{< ref "apps/reference/translations" >}}), and wherever they are defined in the contact forms. Each contact type has its own XForm where it defines the fields that it wants to store/edit for the contact:
+Contact forms are used to create and edit contacts (persons and places). Each contact-type should ideally have two forms; one for creation, and another for editing.
 
-    - /forms/contact/person-create.xml
-    - /forms/contact/person-edit.xml
-    - /forms/contact/clinic-create.xml
-    - /forms/contact/clinic-edit.xml
-    - /forms/contact/health_center-create.xml
-    - /forms/contact/health_center-edit.xml
-    - /forms/contact/district_hospital-create.xml
-    - /forms/contact/district_hospital-edit.xml
-    
-To create the above XForm files it is recommended to use XLSForms. Also, if the contact forms will be similar for all places, you can use a `PLACE_TYPE-create.xlsx` file for creation, a `PLACE_TYPE-edit.xlsx` for editing, and have `cht-conf` generate the individual forms. For this you would also need `forms/contact/places.json`, where you would define the place names. From the XLSForm you can refer to the place type with `PLACE_TYPE`, and the place name with `PLACE_NAME`. You can even use the place type for conditional behaviour in the form. The base files needed would therefore be:
+{{% alert title="Note" %}}
+Prior to version 3.7, CHT Core supported 4 contact types - 3 place types (`clinic`, `health_center`, `district_hospital`) and one person type (`person`). From version 3.7, an unlimited number of place and person types are supported. To learn more about setting up configurable hierarchy, review the [Hierarchy]({{< ref "apps/reference/app-settings/hierarchy" >}}) page.
 
+When "contact_type id" is used below, it is referring to the id of the contact type configured in the `app_settings.json` (if using configurable hierarchy) or the relevant `clinic`, `health_center`, `district_hospital`, or `person` value for the contact in context.
+{{% /alert %}}
 
-    - /forms/contact/person-create.xlsx
-    - /forms/contact/person-edit.xlsx
-    - /forms/contact/PLACE_TYPE-create.xlsx
-    - /forms/contact/PLACE_TYPE-edit.xlsx
-    - /forms/contact/places.json
+These forms are stored in the `forms/contact` sub-folder of the project config directory. The naming convention used should be `<contact_type_id-{create|edit}>.xlsx`.
```

Comment:
This then flows nicely after the opening sentence _Contact forms are used to...._